Articles/Blog/2023-07-BSSwFellows2022.md

```diff
@@ -0,0 +1,238 @@
+
+<div class='short_bio'>
+<p><a href="https://www.sru.edu/c2ac">Nitin</a> is an associate professor in the department of computer science and director of Center for Cybersecurity and Advanced Computing (C2AC) at Slippery Rock University of Pennsylvania. His areas of expertise are scientific computing focusing on performance modeling, robustness and resilience analysis, and cybersecurity.
+</p>
+</div>
+</div>
+
+*Perspectives on the BSSw Fellowship Program:*
+The goal of my work is to aid in finding techniques/methodologies to support secure software development. The BSSw Fellowship Program has enabled me with a platform to reach a broader audience and to network with other researchers in the community with similar goals. With the help of the program, I was able to conduct a tutorial on "Best Practices and Tools For Secure Scientific Software Development" at the SIAM MDS2022 conference, which included components for evaluating design practices for creating secure software, appropriate software processes for managing secure software, threat modeling, and quality assurance testing using both static and dynamic analysis tools. This tutorial provided an opportunity to connect with stakeholders from both academia and industry.
+
+*Advice for new (prospective) BSSw Fellows:*
+I want to say that the BSSw Fellowship Program coordinators are very helpful in providing support while working on delivering the milestones set for the program. I advise that you seek input from the coordinators while setting and narrowing the scope of the milestones. Moreover, participating in annual meetings and other events is highly recommended.
+
+- - -
+
+### Enabling complex scientific computations and efficient use of HPC resources with scientific workflows
+
+BSSw Fellow Karan Vahi brings the use of workflows to the wider scientific community by developing easy-to-understand training materials that examine the workflow lifecycle and challenges associated with various steps such as creation, execution, monitoring, and debugging. Workflows are needed to capture the complex interdependencies between processing steps in data analysis and simulation pipelines as well as the mechanisms to execute those steps reliably and efficiently. Karan's training walks users through how to model existing simulation pipelines into workflows, how to package application code in containers, and how to execute the workflow on HPC resources and distributed computing infrastructure such as Open Science Grid. The training materials also build on existing interactive Jupyter notebooks that guide users on how to develop workflows using [Pegasus](http://pegasus.isi.edu). The training materials are self-guided and well-suited for in-classroom and virtual teaching.
+
+<a href="https://github.com/pegasus-isi/pegasus-usc-tutorial" class="link-row">Pegasus tutorial materials</a>
+<a href="https://www.escience-conference.org/2022/tutorials/pegasus_50_workflows/" class="link-row">Pegasus tutorial</a>
```

Comment:
The text for these two links are virtually identical.  Is it possible to have some text that gives the reader some idea as to the difference between the two tutorials?  Thanks